modules/feegrant/handle_msg.go

```diff
@@ -0,0 +1,47 @@
+package feegrant
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	feegranttypes "github.com/cosmos/cosmos-sdk/x/feegrant"
+	juno "github.com/forbole/juno/v2/types"
+
+	"github.com/forbole/bdjuno/v2/types"
+)
+
+// HandleMsg implements modules.MessageModule
+func (m *Module) HandleMsg(index int, msg sdk.Msg, tx *juno.Tx) error {
```

Comment:
The `index` is not used so we can mark it `_`
```suggestion
func (m *Module) HandleMsg(_ int, msg sdk.Msg, tx *juno.Tx) error {
```